### PR TITLE
Convert 'adblock-banner.js' to ES6

### PR DESCRIPTION
**Pull request by lps88**
ES6 ✨ 

(This wasn't on my list but I am working on a related one, see https://github.com/guardian/frontend/pull/16732)

### CODE REVIEW

File: static/src/javascripts/projects/common/modules/adblock-banner.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+import adblockStickyMessageCoin
+    from 'raw-loader!common/views/commercial/adblock-sticky-message-coin.html';
+
+/**
+ * Message which is shown at the top of the page to the adblock users.
+ * @constructor
+ */
+class AdblockBanner {
+    template: string;
+
```
Comment (rich-nguyen) [inline static/src/javascripts/projects/common/modules/adblock-banner.js]: im suprised `prettier` doesn't fix the gaps here.

File: static/test/javascripts-legacy/spec/common/adblock-banner.spec.js [hunk 2]
```diff
@@ -38,7 +38,7 @@ define([
         it('should be able to render every type of banner', function () {
             ukBanners.forEach(function (banners) {
                 banners.forEach(function (variant) {
-                    new AdblockBanner(variant.template, variant).renderTemplate();
+                    new AdblockBanner.AdblockBanner(variant.template, variant).renderTemplate();
```
Comment (rich-nguyen) [inline static/test/javascripts-legacy/spec/common/adblock-banner.spec.js]: aren't there any other references? Does the module actually get used?

**Review by rich-nguyen — APPROVED** [review]
Good stuff, although the module doesn't appear to be used?

File: static/test/javascripts-legacy/spec/common/adblock-banner.spec.js (hunk 2 above)
Comment (lps88) [inline static/test/javascripts-legacy/spec/common/adblock-banner.spec.js]: Good point 🤔

**Pull request merged**